packages/legend-application-studio/style/components/editor/uml-editor/_uml-element-editor.scss

```diff
@@ -16,6 +16,42 @@
 
 @use 'mixins' as *;
 
+.uml-element-dnd-placeholder {
+  border: 0.2rem dashed var(--color-dark-grey-250);
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0.5rem 1rem 0.5rem 2rem;
+}
+
+.uml-element-dnd-placeholder-container {
+  height: 2.8rem !important;
+  margin: 0.5rem 0;
+}
+
+.uml-element-dnd-placeholder-big-container {
+  padding: 2rem 1rem;
+}
+
+.uml-element-dnd-placeholder-big {
+  border: 0.2rem dashed var(--color-dark-grey-250);
+  padding: 3rem 0.5rem;
+  margin: 0;
+}
+
+.uml-element-dnd-name {
+  position: fixed;
+  display: inline-flex;
```

Comment:
A bit, neaty picky here, but let's use `<div>` instead of `<span>` and use `@include flexCenter` for these. Can we?

Reply:
yep! 👍